sregistry/main/registry/__init__.py

```diff
@@ -87,17 +87,20 @@ def _add_https(self, q):
 
 
     def _update_secrets(self):
-        '''update secrets will take a secrets credential file
-           either located at .sregistry or the environment variable
-           SREGISTRY_CLIENT_SECRETS and update the current client 
-           secrets as well as the associated API base.
+        '''update secrets will use the SREGISTRY_REGISTRY_BASE to determine which iAPI base url
+           to use for the registry. If not present, the .sregistry secrets credential file
```

Comment:
Does `iAPI` mean something special? I think you can just leave it out to say "which base url"

Reply:
That's a typo... fixing...